server/config/rateLimiter.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+const rateLimit = require('express-rate-limit')
```

Comment:
Should we move this file to middleware? Looks like a middleware cus i saw this.router.post('', apiLimiter, this.create.bind(this))